code/modules/clothing/under/miscellaneous.dm

```diff
@@ -728,3 +728,12 @@
 	icon_state = "hostanclothes"
 	item_state = "hostanclothes"
 	item_color = "hostanclothes"
+
+/obj/item/clothing/under/rank/iaa

```

Comment:
Didn't we have a whole discussion about how the IAA icons exist in the code already?